.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: Docker CI build and run
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build docker image
+      run: |
+        docker build . --file ./.docker/Dockerfile --tag swedbankpay/jekyll-plantuml:ci
+
+    - name: Check workspace
+      run: |
+        ls -la ${{ github.workspace }}/test/jekyll-plantuml
+
+    - name: Run docker image
+      run: |
+        docker run --tty --volume ${{ github.workspace }}/test/jekyll-plantuml:/srv/jekyll swedbankpay/jekyll-plantuml:ci "jekyll build"
```

Should `"jekyll build"` be quoted here when it's not in `push_image_and_stages` from `build_and_publish.yml`?

```suggestion
        docker run --tty --volume ${{ github.workspace }}/test/jekyll-plantuml:/srv/jekyll swedbankpay/jekyll-plantuml:ci jekyll build
```